tests/integration/test_interaction.py

```diff
@@ -581,3 +581,138 @@ def test_keyboard_help(self):
+        self._assert_zone_align_item(4, 'Zone Invalid Align', 'start')
+        self._assert_zone_align_item(5, 'Zone Invalid Align', 'start')
+
+    def test_zone_align_invalid_key(self):
+        """
+        Test items placed in a zone with an invalid align setting, via keyboard.
+        """
+        self._assert_zone_align_item(3, 'Zone Invalid Align', 'start', Keys.RETURN)
+        self._assert_zone_align_item(4, 'Zone Invalid Align', 'start', Keys.RETURN)
+        self._assert_zone_align_item(5, 'Zone Invalid Align', 'start', Keys.RETURN)
+
+    def test_zone_align_left(self):
+        """
+        Test items placed in a zone with an left align setting.
+        """
+        self._assert_zone_align_item(6, 'Zone Left Align', 'left')
+        self._assert_zone_align_item(7, 'Zone Left Align', 'left')
+        self._assert_zone_align_item(8, 'Zone Left Align', 'left')
+
+    def test_zone_align_left_key(self):
+        """
+        Test items placed in a zone with an left align setting, via keyboard.
+        """
+        self._assert_zone_align_item(6, 'Zone Left Align', 'left', Keys.RETURN)
+        self._assert_zone_align_item(7, 'Zone Left Align', 'left', Keys.RETURN)
+        self._assert_zone_align_item(8, 'Zone Left Align', 'left', Keys.RETURN)
+
+    def test_zone_align_right(self):
+        """
+        Test items placed in a zone with a right align setting.
+        """
+        self._assert_zone_align_item(9, 'Zone Right Align', 'right')
+        self._assert_zone_align_item(10, 'Zone Right Align', 'right')
+        self._assert_zone_align_item(11, 'Zone Right Align', 'right')
+
+    def test_zone_align_right_key(self):
+        """
+        Test items placed in a zone with an right align setting, via keyboard.
+        """
+        self._assert_zone_align_item(9, 'Zone Right Align', 'right', Keys.RETURN)
+        self._assert_zone_align_item(10, 'Zone Right Align', 'right', Keys.RETURN)
+        self._assert_zone_align_item(11, 'Zone Right Align', 'right', Keys.RETURN)
+    '''
+    FIXME Erratic bug in place_item(action_key=None) when there's too many zones?
+    Works consistently from keyboard and manual testing.
+    def test_zone_align_center(self):
+        """
+        Test items placed in a zone with an center align setting.
+        """
+        self._assert_zone_align_item(12, 'Zone Center Align', 'center')
+        self._assert_zone_align_item(13, 'Zone Center Align', 'center')
+        self._assert_zone_align_item(14, 'Zone Center Align', 'center')
+    '''
+    def test_zone_align_center_key(self):
+        """
+        Test items placed in a zone with an center align setting, via keyboard.
+        """
+        self._assert_zone_align_item(12, 'Zone Center Align', 'center', Keys.RETURN)
+        self._assert_zone_align_item(13, 'Zone Center Align', 'center', Keys.RETURN)
+        self._assert_zone_align_item(14, 'Zone Center Align', 'center', Keys.RETURN) 
+
```

Comment:
@pomegranited These tests look good, but you could use `ddt` to DRY them out a little. Rough sketch:

```python
@ddt
class ZoneAlignInteractionTest(InteractionTestBase, BaseIntegrationTest):
    ACTION_KEYS = (None, Keys.RETURN, Keys.SPACE, Keys.CONTROL+'m', Keys.COMMAND+'m')
    # ...
    @data(
        ([3, 4, 5], "Zone Invalid Align"),
        ([6, 7, 8], "Zone Left Align"),
        ([9, 10, 11], "Zone Right Align"),
        ([12, 13, 14], "Zone Center Align"),
    )
    @unpack
    def test_zone_align(self, items, zone):
        for item in items:
            for action_key in self.ACTION_KEYS:
                self.assert_zone_align_item(item, zone, action_key)
```

Including all possible keyboard shortcuts for dropping an item in `ACTION_KEYS` might be overkill. It would probably also be fine to just do `ACTION_KEYS = (None, Keys.RETURN)`, I'll leave that up to you.